jack/block-type-preference (#399)
vellum-ai/vellum-client-generator: fern/docs/content/change-log/2024-09.mdx

title: Changelog | September, 2024
---

## Default Block Type Preference

_September 4th, 2024_

> You can now set a default Block type for use throughout the app. Whenever you see the "Add Block" or "Add Message" options in a Prompt Editor, your preferred Block type will be used.

Reviewer: ```suggestion
You can now set a default Block type to use when defining Prompts in Vellum. Whenever you see the "Add Block" or "Add Message" options in a Prompt Editor, your preferred Block type will be used.
```
